projects/fusion-ui/src/lib/directives/click-outside/click-outside.directive.ts

```diff
@@ -43,6 +50,16 @@ export class ClickOutsideDirective implements OnInit, OnDestroy {
         }
     }
 
+    private isClickInsideByCoordinates(event: MouseEvent): boolean {
```

Comment:
add comment explain what this is do

Reply:
Added comment on @Input() definition - Line#20